AppCUI/src/Controls/Desktop.cpp

```diff
@@ -60,6 +59,22 @@ bool Desktop::OnKeyEvent(AppCUI::Input::Key keyCode, char16_t /*UnicodeChar*/)
         GoToNextWindow(Members, -1);
         return true;
     }
-
+    // check cntrols hot keys
+    if ((((unsigned int) keyCode) & (unsigned int) (Key::Shift | Key::Alt | Key::Ctrl)) == ((unsigned int) Key::Alt))
+    {
+        auto* b = Members->Controls;
+        auto* e = b + Members->ControlsCount;
+        while (b < e)
+        {
+            ControlContext* winMembers = (ControlContext*) (*b)->Context;
```

Comment:
acolo unde se casteaza explicit in partea dreapta cred ca se poate folosi un auto
const auto winMembers = reinterpret_cast<ControlContext*>( (*b)->Context);